src/components/content-tab-overview.tsx

```diff
@@ -164,8 +164,12 @@ function ShortcutsSection( { selectedSite }: Pick< ContentTabOverviewProps, 'sel
 			label: editorConfig.label,
 			className: 'text-nowrap',
 			icon: code,
-			onClick: () => {
-				getIpcApi().openURL( editorConfig.url( selectedSite.path ) );
+			onClick: async () => {
+				if ( isMac() ) {
+					await getIpcApi().openAppAtPath( editorConfig.bundleId, selectedSite.path );
+				} else {
+					getIpcApi().openURL( editorConfig.url( selectedSite.path ) );
+				}
```

Comment:
```tsx
				await getIpcApi().openAppAtPath( editor, selectedSite.path );
```
Just a suggestion, but I would consider exposing a single IPC function that internally determines which action to take depending on the current platform. We can't pass the `editorConfig` object, but we can pass the `editor` key and import `supportedEditorConfig` in `ipc-handlers.ts`

Reply:
Thanks, this is a good point. I'll make these changes in the PR fixing the same issue on Windows.